pkg/resources/utilities.go

```diff
@@ -0,0 +1,30 @@
+// Package resources contains utility functions which are useful for both mutator for validator
```

Comment:
These functions fit well in the pkg/admission package.
And the names could then be updated to ResponseAllowed and ResponseBadRequest

Reply:
good idea! 
fixed